modules/server/src/main/ts/BedrockManual.ts

```diff
@@ -26,6 +26,7 @@ export const go = (settings: BedrockSettings): void => {
       coverage: settings.coverage,
       overallTimeout: settings.overallTimeout,
       singleTimeout: settings.singleTimeout,
+      polyfills: settings.polyfills,
```

Comment:
Do we use all of `settings` here, or just a subset?

Reply:
All of it is used in various places of the code IIUC.

Reply:
I was just wondering if we could use `...settings` in here.

Reply:
Ahh I see, then probably... we'd still need the few extra things not in `settings`. Most of this is still hanging around from before we made the conversion to TypeScript, so it's probably just that we hadn't thought of switching to that.

I'll give it a shot and see how it goes anyways :shrug:

Reply:
Done. It does cause other settings to be pulled in, in some cases, but that should be an issue and makes it easier to add new settings.